node/src/chain_spec.rs

```diff
@@ -184,6 +185,18 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
     .with_id("nh_testnet")
     .with_protocol_id("tzen")
     .with_chain_type(ChainType::Live)
+    .with_boot_nodes(vec![
+        "/dns/nh-bootnode-1.horizenlabs.io/tcp/443/wss/p2p/12D3KooWNhvf6iSowraUY4tZnjpNZXEe85oy9zDWYRKFBnWivukc"
+        .parse()
+        .expect("MultiaddrWithPeerId"),
+        "/dns/nh-bootnode-2.horizenlabs.io/tcp/443/wss/p2p/12D3KooWEjVadU1YWyfDGvyRXPbCq2rXhzJtXaG4RxJZBkGE9Aug"
+        .parse()
+        .expect("MultiaddrWithPeerId"),
+    ])
+    .with_telemetry_endpoints(
+		TelemetryEndpoints::new(vec![("wss://nh-telemetry.horizenlabs.io/submit".to_string(), 1)])
```

Comment:
I'd suggest replacing the verbosity level value with its named constant.
```suggestion
		TelemetryEndpoints::new(vec![("wss://nh-telemetry.horizenlabs.io/submit".to_string(), sc_telemetry::CONSENSUS_INFO)])
```

Reply:
Ok I agree on both notes.
I have replaced `1` with `sc_telemetry::CONSENSUS_INFO` and now `STAGING_TELEMETRY_URL` is set and used (done similarly also for bootnodes connection strings).